JS/fluentRevealNavbar.uc.js

```diff
@@ -139,6 +167,13 @@
         ? el
         : el.querySelector(".toolbarbutton-badge-stack") ||
           el.querySelector(".toolbarbutton-icon");
+      if (!area) area = el;
```

Comment:
Is this just for the urlbar? Would prefer to handle it explicitly. That way if something unintended makes it into here, there will be an error in the log as `area` will not exist.
```suggestion
      if (!area && el.id == "urlbar-background") area = el;
```

Reply:
Yeah, just for url bar. I think this one would be a bit more concise, while not allowing for unintended elements? 

```
let area  =
       isBookmark ? el
        : el.id == "urlbar-background" ? el
        : el.querySelector(".toolbarbutton-badge-stack") ||
          el.querySelector(".toolbarbutton-icon");
```

Reply:
Nope, because there's an [eslint rule](https://eslint.org/docs/latest/rules/no-nested-ternary) that disallows nested ternary expressions. Another reason you'll want to set up the npm package in the repo. It'll be tedious trying to resolve all the linting and formatting issues manually.